app/controllers/concerns/state_file/eligibility_offboarding_concern.rb

```diff
@@ -20,7 +20,7 @@ def next_path
     end
 
     def from_path_params
-      [:us_state, :return_to_review].each_with_object({}) do |key, path_params|
+      [:return_to_review].each_with_object({}) do |key, path_params|
         path_params[key] = params[key] if params[key].present?
       end
     end
```

Comment:
minor thing but this doesn't seem like it needs to be its own method anymore? maybe?